src/materials/fullscreen/BlendMaterial.js

```diff
@@ -14,6 +14,7 @@ export class BlendMaterial extends MaterialBase {
 				target1: { value: null },
 				target2: { value: null },
 				opacity: { value: 1.0 },
+				t2conversion: { value: false }
```

Comment:
What does "t2conversion" mean?

Reply:
I needed a way to apply colorspace conversion to one of the inputs. In this case "target2". "t2conversion" is just short for "target2 color conversion"